lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openstack-network_manager-cloud_subnet.rb

```diff
@@ -1,4 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceManageIQ_Providers_Openstack_NetworkManager_CloudSubnet < MiqAeServiceCloudSubnet
+    expose :delete_cloud_subnet_queue
```

Comment:
shouldn't it be enough to just use the API calls? I thought we don't want to expose more methods this way.